### PR TITLE
Update outlook_analyzer.py w/ clean code

### PR DESCRIPTION
**Pull request by tingjlee**
Just added the clean code at the end of everything.

### CODE REVIEW

File: outlook_analyzer.py [hunk 1]
```diff
@@ -519,6 +521,19 @@ def main():
         print("Some errors occurred during execution:")
         for item in ERROR_LIST:
             print(item)
+            
+  # Checks if user wants to clean their Temp directory
+    while True:
+        dir_option = input('Would you like to clean your directory? (y/n):')
+        if dir_option == 'y' or dir_option == 'Y':
+            for item in DIR_LIST:
```
Comment (cplummer-uwm) [inline outlook_analyzer.py]: You should create a new folder in C:\WINDOWS\Temp for outlookanalyzer and only clean the files in that directory. If you clean all the files with .txt/.png/.jpg, you might be breaking some other program that is using those files.

File: outlook_analyzer.py [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 from datetime import timedelta
 from datetime import date
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
-import sys
 
 ERROR_LIST = []
 TIME_STR = time.strftime("%Y%m%d-%H%M%S")
 TEMP_DIR = "C:\WINDOWS\Temp"
+DIR_LIST = os.listdir(TEMP_DIR) # Puts Temp directory as a list
```
Comment (cplummer-uwm) [inline outlook_analyzer.py:26]: You should update TEMP_DIR above to be a folder unique to us and create it if it doesn't exist.

**Review by cplummer-uwm — CHANGES_REQUESTED** [review]

File: outlook_analyzer.py (hunk 1 above)
Comment (lisashub) [reply outlook_analyzer.py]: My bad, if you got an email ping with my previous comment. Thought for a hot second that using the TEMP directory was breaking something for Ting, but apparently not?

Comment (cplummer-uwm) [reply outlook_analyzer.py]: Not sure yet where the "issue" is or even if there is a problem. Using a temp location is a best practices for temp files. See https://softwareengineering.stackexchange.com/questions/314796/should-temporary-files-be-saved-to-tmp-or-the-current-working-directory

Comment (lisashub) [reply outlook_analyzer.py]: Yes, I'd mentioned in my previous comment that I _thought_ there was an issue, but apparently there isn't one, haha. I'd made a previous comment (now deleted) based on my first impression. My existing comment is a heads up that the now deleted comment was made in error; I know GitHub sends email summaries.

**Comment by cplummer-uwm** [comment]
Closing this PR since this code was integrated into my branch. Thanks for the ideas.

**Comment by cplummer-uwm** [comment]
Closing this PR since this code was integrated into my branch. Thanks for the ideas.

**Pull request closed without merging**